.github/workflows/build.yml

```diff
@@ -202,8 +202,8 @@ jobs:
           exit 1
         fi
 
-        # Check that commit contains ':'
-        if commit_titles |  grep -E -v '^([.a-z]+, )*[.a-z]+: [a-z]'; then
+        # Check that commit contains ':' (and no '!')
+        if commit_titles | grep -E -v '^([.a-z\-]+, )*[.a-z\-]+: [a-z\-]'; then
```

Comment:
```suggestion
        if commit_titles | grep -E -v '^([.a-z\-]+, )*[.a-z\-]+: [a-z]'; then
```

Reply:
Discussion move in https://github.com/osrd-project/osrd/pull/6528#discussion_r1474079699